### ISSUE TITLE
Update 'pesticide' to 'insecticide load'

### ISSUE THREAD

**Issue by fungjj92** [project-icp/bee-pollinator-app#488]
Change "Pesticide quality" to "Insecticide load". This will also mean changing the Pesticide naming convention to Insecticide, renaming the underlying COGs and updating the `vrt` accordingly. Confirm with the client that this is what they want and can update data correctly in the future.

Note that renaming files on s3 will break staging. Create duplicates of pesticide data, renamed to insecticide, and new corresponding `vrts`. Be sure to delete stale data off s3 once this work is merged and deployed to staging.

Ensure production data, if we have production at the time, is updated as necessary.

**Comment by dboyer** [comment]
Confirmed that we can change the naming convention to Insecticide load. Penn State will adjust the name of any insecticide rasters they upload in the future.

**Comment by mmcfarland** [comment]
I'm curious if this can be done by simply updating the label for Pesticide without having to update all of our raster sources and scripts? Perhaps with a comment if the concern is that it will be confusing in the future for which dataset represents Insecticide?
